tests/cover/test_direct_strategies.py

```diff
@@ -225,6 +227,29 @@ def test_produces_valid_examples_from_args(fn, args):
     fn(*args).example()
 
 
+def test_build_class_with_target_kwarg():
+    NamedTupleWithTargetField = collections.namedtuple('Something', ['target'])
+    ds.builds(NamedTupleWithTargetField, target=ds.integers())
```

Comment:
Need to add a `.example()` on the end, or there could be a lazy error lurking 😄

Reply:
oh, right!